src/FirePokemon.java

```diff
@@ -0,0 +1,48 @@
+public class FirePokemon extends Pokemon {
+
+//    Could place an arraylist for all fire type pokemons??
```

Comment:
Je kunt die arraylist hier neer zetten, misschien static, maar ik denk dat het beter is om die list in Main te zetten (zoals ik ook al in een eerdere omment zei)